lib/friendly_shipping/services/ship_engine/parse_address_validation_response.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module FriendlyShipping
+  module Services
+    class ShipEngine
+      class ParseAddressValidationResponse
+        extend Dry::Monads::Result::Mixin
+
+        class << self
+          # @param [FriendlyShipping::Request] request
+          # @param [FriendlyShipping::Response] response
+          # @return [Success<ApiResult>, Failure<ApiFailure>]
```

Comment:
Should this be Success<ApiResult<Array<Physical::Location>>>?

Reply:
Sure. More detail won't hurt.